typescript/.changeset/lovely-vans-knock.md

```diff
@@ -0,0 +1,5 @@
+---
+"@coinbase/agentkit": patch
```

Comment:
nit: This introduces new behavior, so it should be a minor. Feel free to edit this file directly, no need to rerun the changesets command.